src/pages/chat/Chat.tsx

```diff
@@ -47,31 +100,51 @@ const Chat = () => {
         >
           <ListOfUsersWithSearch
             closeDrawer={closeDrawer}
-            isSelectedChat={isSelectedChat}
             listOfChats={listOfChats}
-            setIsSelectedChat={setIsSelectedChat}
+            selectedChat={selectedChat}
+            setSelectedChat={setSelectedChat}
           />
         </AppDrawer>
       )}
       <Allotment defaultSizes={isMobile ? [1] : [25, 75]}>
         {!isMobile && (
           <Allotment.Pane minSize={250} preferredSize={350}>
             <ListOfUsersWithSearch
-              isSelectedChat={isSelectedChat}
               listOfChats={listOfChats}
-              setIsSelectedChat={setIsSelectedChat}
+              selectedChat={selectedChat}
+              setSelectedChat={setSelectedChat}
             />
           </Allotment.Pane>
         )}
         <Allotment.Pane minSize={350}>
-          <>
-            {isMobile && (
-              <IconButton onClick={openChatsHendler}>
-                <MenuIcon />
-              </IconButton>
+          <Box sx={styles.chatContent(selectedChat)}>
+            {!selectedChat ? (
+              selectChatChip
+            ) : (
+              <>
+                <ChatHeader
```

Comment:
i thin that the second part of the condition also should be moved above in separate variable to be more readable

Reply:
fixed